src/content/getting-started/designers/designers.md

```diff
@@ -3,34 +3,32 @@ label: Getting Started
 title: Designers
 ---
 
-<page-intro>The **Carbon Design Kit** is a living, breathing document that contains all of our visual assets (components, iconography, color palettes, grids, etc) aimed to help you work faster, with confidence and thoughtfulness.</page-intro>
+<page-intro>This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon.</page-intro>
```

Comment:
Not sure if I like a direct reference to the site on the site but not that big of a thing.